packages/story-editor/src/types/story.ts

```diff
@@ -36,6 +37,12 @@ interface FeaturedMedia {
   isExternal: boolean;
 }
 
+export type ElementSaveData = Element;
+
+export interface PageSaveData extends Omit<Page, 'elements'> {
+  elements: ElementSaveData[];
+}
+
```

Comment:
Should we remove this again now that `ElementSaveData` is not different anymore?